scripts/install-cni.sh

```diff
@@ -110,15 +115,20 @@ else
   cni_spec=${cni_spec//@cniBandwidthPlugin/}
 fi
 
-token=$(</var/run/secrets/kubernetes.io/serviceaccount/token)
 host=${KUBERNETES_SERVICE_HOST}
 # If host contains a colon (:), it is an IPv6 address, hence needs wrapping
 # with [..].
 if [[ "${host}" =~ : ]]; then
   host="[$host]"
 fi
-node_url="https://$host:${KUBERNETES_SERVICE_PORT}/api/v1/nodes/${HOSTNAME}"
-response=$(curl -k -s -H "Authorization: Bearer $token" "$node_url")
+node_url="https://$host:${KUBERNETES_SERVICE_PORT}/api/v1/watch/nodes/${HOSTNAME}"
+for _ in {1..3}; do
```

Comment:
nit: 1..3 is a magic number. make it a named const variable that says this is the number of retries.

Reply:
Done.